hermes-api/src/main/java/pl/allegro/tech/hermes/api/OfflineRetransmissionRequest.java

```diff
@@ -24,7 +26,7 @@ public class OfflineRetransmissionRequest {
     );
     private static final Logger logger = LoggerFactory.getLogger(OfflineRetransmissionRequest.class);
 
-    @NotEmpty
+    private final String sourceViewPath;
```

Comment:
**suggestion:** Maybe lets stick with a `table` terminology as the retransmission can work both on tables and views, and view is some kind `table`